client/src/app/site/motions/modules/motion-poll/motion-poll-detail/motion-poll-detail.component.html

```diff
@@ -1,9 +1,9 @@
 <os-head-bar [goBack]="true" [nav]="false">
     <div class="title-slot">
-        <h2 *ngIf="poll">{{ 'Motion' | translate }} {{ poll.getContentObject().numberOrTitle }}</h2>
+        <h2 *ngIf="poll">{{ 'Motion' | translate }} {{ poll.motion.identifierOrTitle }}</h2>
```

Comment:
When I wanted to see my results, this error appears:
`TypeError: Cannot read property "identifierOrTitle" of undefined`

Reply:
yep had too many changes here